bazaarvoice/emodb
# Pd 171750 fix facade related postman tests

## What Are We Doing Here?

Here we are fixing facade related tests in facade and import stream arbitrary size endpoints postman tests

## How to Test and Verify

1. Run this branch on CI job for running postman tests
2. Verify that facade and imports stream arbitrary size tests are passed


## Risk
more failed tests and less tested application as a result 

### Level 

`Low`

### Required Testing

`Manual`

### Risk Summary
Most of the tests have been failed before so there is only risk that more facade related tests are broken by this fix


## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
